spec/appendices.md

```diff
@@ -14,17 +14,31 @@ host environments, their serializations and resource formats,
 that might be sufficient to prevent most problems.
 However, MessageFormat itself does not supply such a restriction.
 
-MessageFormat _messages_ permit nearly all Unicode code points,
-with the exception of surrogates, 
+MessageFormat _messages_ permit nearly all Unicode code points
 to appear in _literals_, including the text portions of a _pattern_.
 This means that it can be possible for a _message_ to contain invisible characters
-(such as bidirectional controls, 
-ASCII control characters in the range U+0000 to U+001F,
+(such as bidirectional controls, ASCII control characters in the range U+0000 to U+001F,
 or characters that might be interpreted as escapes or syntax in the host format)
 that abnormally affect the display of the _message_
 when viewed as source code, or in resource formats or translation tools,
 but do not generate errors from MessageFormat parsers or processing APIs.
 
+> [!IMPORTANT]
+> _Text_ and _quoted literals_ allow unpaired surrogate code points
+> (`U+D800` to `U+DFFF`).
```

Comment:
How/why is this important to note as a security consideration?

Reply:
I previously suggested the same thing:

> I... don't think this is a security consideration? Maybe move this note to the section on message, just after the note that begins:
>
>>    This syntax is designed to be embeddable into many different programming languages...

I think that location is a good one because we put a whole wodge of notes there that are "read once and forget" about the content of messages.